.buildkite/pull-requests.json

```diff
@@ -15,6 +15,19 @@
       "trigger_comment_regex": "(run\\W+elasticsearch-ci.+)|(^\\s*((buildkite|@elastic(search)?machine)\\s*)?test\\s+this(\\s+please)?)",
       "cancel_intermediate_builds": true,
       "cancel_intermediate_builds_on_comment": false
+    },
+    {
+      "enabled": true,
+      "pipeline_slug": "elasticsearch-performance-esbench-pr",
+      "allow_org_users": true,
+      "allowed_repo_permissions": [
+        "admin",
+        "write"
+      ],
+      "set_commit_status": false,
+      "build_on_commit": false,
+      "build_on_comment": true,
+      "trigger_comment_regex": "^\\s*((buildkite|@elastic(search)?machine)\\s*)?benchmark this with (?<benchmark>\\w+)(\\s+please)?"
```

Comment:
The `((buildkite|@elastic(search)?machine)\\s*)` group is optional. Perhaps we should make it mandatory to reduce a chance of unnecessary trigger?

Reply:
I've narrowed it down in [c64371c](https://github.com/elastic/elasticsearch/pull/132717/commits/c64371c091cc07ca09b64459d6afa310f038b3a9)